js/index.js

```diff
@@ -1,6 +1,34 @@
+console.log("I'm ready!")
+
 // Iteration 1: Names and Input
-//
+let hacker1 = "Sergio"
+console.log("The driver's name is " + hacker1)
```

Comment:
Esta manera está bien, pero para interpolar variables se suele usar emplear así ----> 
`console.log(`The driver's name  is ${hacker1}.` )`